### PR TITLE
Make save as use the scroll menu

### LINKED ISSUE #81: Use the scroll menu to select a directory for save as
should be very similar to renaming.

Open scroll menu, `show_files=False`. Once a directory is shown, ask for the name of the file. do not allow `/` or `\` since the directory should be chosen from the scroll menu, not typed out.

This will also fix the bug of not properly marking a file as removed or deleted on windows if the file was saved with a forward slash (/).

### PR DESCRIPTION
**Pull request by SharpBit**
address issue #81

### CODE REVIEW

**Comment by SharpBit** [comment]
Merge PR #80 before this one.

**Comment by CupOfGeo** [comment]
when i click save on a new file i get a bad dialog box 
image

**Comment by CupOfGeo** [comment]
naturally i use poop pee and but as all my test strings

**Comment by SharpBit** [comment]
> when i click save on a new file i get a bad dialog box
> image

"fixed" (for user experience) in 9bf1c8045454c14c5d0213635969159fc206f726 (#80)

File: src/navigation/menu_bar.py [hunk 1]
```diff
@@ -114,48 +114,50 @@ async def coroutine(self: MenuNav) -> None:
 
             If the path entered is a valid file name, save the current note at that path.
             """
+            dialog = ScrollMenuDialog(
+                title="Save As",
+                text="Choose the location of the file.",
+                directory=self.application_state.current_dir,
+                show_files=False,
+            )
+            directory = await self.show_dialog_as_float(dialog)
```
Comment (CupOfGeo) [inline src/navigation/menu_bar.py:124]: wondering if we can show what the selected on the left side of the scroll which would be done in ScrollMenuDialog

**Review by CupOfGeo — APPROVED** [review]
It works

**Pull request merged**